app/src/utils/utils.ts

```diff
@@ -118,3 +128,31 @@ export function displayResults(
   const end = Math.min(page * perPage, numFound);
   return `${start}-${end} of ${numFound}`;
 }
+
+export const createQueryStringFromObject = (paramObj) => {
+  const params = new URLSearchParams();
+  Object.keys(paramObj).map((name, value) => {
+    params.set(
+      name.toString(),
+      name === "page" ? paramObj[name].toString() : paramObj[name]
```

Comment:
This specific check, along with `DEFAULT_PAGE_NUM.toString()`, are minor code flags. I know you're making `DEFAULT_PAGE_NUM` a number since that's what the DS uses, but for consistency and to make these functions easier, a string is a better choice. Then in `collectionPage.tsx`, you can use it as `Number(DEFAULT_PAGE_NUM)`. Then it's only changed once and these functions will be easier to maintain.